### PR TITLE
build,scripts: Fix package downloading and availability check from bootstrap script

### PR DESCRIPTION
**Pull request by TeddyAndrieux**
- Fix the common script function to check the availability of packages
- Allow downloading packages required by our build packages
- Do no longer explicitly require  `sos` package (since it's a required of `metalk8s-sosreport`)
- Remove duplicated packages downloaded

---

<details><summary>difference on the ISO with this PR</summary>

ISO size
```console
-rw-r--r--. 1 root root 2.1G Oct 11 14:45 metalk8s-before.iso
-rw-r--r--. 1 root root 1.9G Oct 12 07:59 metalk8s-now.iso
```

Number of packages
```console
$ find before/packages/ -name "*.rpm" | wc -l
81
$ find now/packages/ -name "*.rpm" | wc -l
56
```

List of RPM before this PR
```console
$ find . -name "*.rpm"
./redhat/7/metalk8s-epel-el7/libnetfilter_cthelper-1.0.0-4.el7.x86_64.rpm
./redhat/7/metalk8s-epel-el7/libsodium-1.0.18-1.el7.x86_64.rpm
./redhat/7/metalk8s-epel-el7/openpgm-5.2.122-2.el7.x86_64.rpm
./redhat/7/metalk8s-epel-el7/python36-asn1crypto-0.24.0-7.el7.noarch.rpm
./redhat/7/metalk8s-epel-el7/python36-cffi-1.9.1-3.el7.x86_64.rpm
./redhat/7/metalk8s-epel-el7/python36-cryptography-2.3-2.el7.x86_64.rpm
./redhat/7/metalk8s-epel-el7/python36-distro-1.5.0-1.el7.noarch.rpm
./redhat/7/metalk8s-epel-el7/python36-idna-2.10-1.el7.noarch.rpm
./redhat/7/metalk8s-epel-el7/python36-jinja2-2.11.1-1.el7.noarch.rpm
./redhat/7/metalk8s-epel-el7/python36-m2crypto-0.35.2-5.el7.x86_64.rpm
./redhat/7/metalk8s-epel-el7/python36-ply-3.9-2.el7.noarch.rpm
./redhat/7/metalk8s-epel-el7/python36-psutil-5.6.7-1.el7.x86_64.rpm
./redhat/7/metalk8s-epel-el7/python36-pycparser-2.14-2.el7.noarch.rpm
./redhat/7/metalk8s-epel-el7/python36-pysocks-1.6.8-7.el7.noarch.rpm
./redhat/7/metalk8s-epel-el7/python36-PyYAML-3.13-1.el7.x86_64.rpm
./redhat/7/metalk8s-epel-el7/python36-requests-2.14.2-2.el7.noarch.rpm
./redhat/7/metalk8s-epel-el7/python36-rpm-4.11.3-10.el7.x86_64.rpm
./redhat/7/metalk8s-epel-el7/python36-six-1.14.0-3.el7.noarch.rpm
./redhat/7/metalk8s-epel-el7/python36-typing-3.5.2.2-4.el7.noarch.rpm
./redhat/7/metalk8s-epel-el7/python36-urllib3-1.25.6-2.el7.noarch.rpm
./redhat/7/metalk8s-kubernetes-el7/14083ac8b11792469524dae98ebb6905b3921923937d6d733b8abb58113082b7-kubernetes-cni-1.1.1-0.x86_64.rpm
./redhat/7/metalk8s-kubernetes-el7/87b6290108e1c8429f6b50895a6376d92f2cd730db2925763c470f32e98b2e2f-kubelet-1.24.6-0.x86_64.rpm
./redhat/7/metalk8s-kubernetes-el7/ad9e22b62da3b131b3b91fd2caee123cf298133bc58edd6733d7676b697fcd40-kubectl-1.24.6-0.x86_64.rpm
./redhat/7/metalk8s-kubernetes-el7/e382ead81273ab8ebcddf14cc15bf977e44e1fd541a2cfda6ebe5741c255e59f-cri-tools-1.25.0-0.x86_64.rpm
./redhat/7/metalk8s-saltstack-el7/python36-3.6.8-2.el7.x86_64.rpm
./redhat/7/metalk8s-saltstack-el7/python36-chardet-3.0.4-12.el7.noarch.rpm
./redhat/7/metalk8s-saltstack-el7/python36-contextvars-2.4-1.el7.noarch.rpm
./redhat/7/metalk8s-saltstack-el7/python36-immutables-0.14-1.el7.x86_64.rpm
./redhat/7/metalk8s-saltstack-el7/python36-libs-3.6.8-2.el7.x86_64.rpm
./redhat/7/metalk8s-saltstack-el7/python36-markupsafe-0.23-4.el7.x86_64.rpm
./redhat/7/metalk8s-saltstack-el7/python36-msgpack-0.6.2-2.el7.x86_64.rpm
./redhat/7/metalk8s-saltstack-el7/python36-pycurl-7.43.0-8.el7.x86_64.rpm
./redhat/7/metalk8s-saltstack-el7/python36-pyOpenSSL-17.3.0-2.el7.noarch.rpm
./redhat/7/metalk8s-saltstack-el7/python36-setuptools-39.2.0-4.el7.noarch.rpm
./redhat/7/metalk8s-saltstack-el7/python36-zmq-17.0.0-6.el7.x86_64.rpm
./redhat/7/metalk8s-saltstack-el7/salt-3002.9-1.el7.noarch.rpm
./redhat/7/metalk8s-saltstack-el7/salt-minion-3002.9-1.el7.noarch.rpm
./redhat/7/metalk8s-saltstack-el7/yum-utils-1.1.31-29.el7.noarch.rpm
./redhat/7/metalk8s-saltstack-el7/zeromq-4.1.4-7.el7.x86_64.rpm
./redhat/7/metalk8s-scality-el7/x86_64/containerd-1.6.8-1.el7.x86_64.rpm
./redhat/7/metalk8s-scality-el7/x86_64/metalk8s-sosreport-124.0-2.el7.x86_64.rpm
./redhat/8/metalk8s-epel-el8/libsodium-1.0.18-2.el8.x86_64.rpm
./redhat/8/metalk8s-epel-el8/libunwind-1.3.1-3.el8.x86_64.rpm
./redhat/8/metalk8s-epel-el8/openpgm-5.2.122-21.el8.x86_64.rpm
./redhat/8/metalk8s-epel-el8/python3-m2crypto-0.35.2-5.el8.x86_64.rpm
./redhat/8/metalk8s-epel-el8/python3-msgpack-0.6.2-1.el8.x86_64.rpm
./redhat/8/metalk8s-epel-el8/python3-zmq-19.0.0-1.el8.x86_64.rpm
./redhat/8/metalk8s-epel-el8/zeromq-4.3.4-2.el8.x86_64.rpm
./redhat/8/metalk8s-kubernetes-el8/029bc0d7b2112098bdfa3f4621f2ce325d7a2c336f98fa80395a3a112ab2a713-kubernetes-cni-0.8.6-0.x86_64.rpm
./redhat/8/metalk8s-kubernetes-el8/07433570e95a2782cc127e659fe6df434db7f88805e2aed6067768d2f32cb809-cri-tools-1.24.2-0.x86_64.rpm
./redhat/8/metalk8s-kubernetes-el8/0c923b191423caccc65c550ef07ce61b97f991ad54785dab70dc07a5763f4222-kubernetes-cni-0.3.0.1-0.07a8a2.x86_64.rpm
./redhat/8/metalk8s-kubernetes-el8/14083ac8b11792469524dae98ebb6905b3921923937d6d733b8abb58113082b7-kubernetes-cni-1.1.1-0.x86_64.rpm
./redhat/8/metalk8s-kubernetes-el8/14bfe6e75a9efc8eca3f638eb22c7e2ce759c67f95b43b16fae4ebabde1549f3-cri-tools-1.13.0-0.x86_64.rpm
./redhat/8/metalk8s-kubernetes-el8/29e7806d1d54cc0eea2963f4ab276778526538816c88c963ece7d1a05fd80792-cri-tools-1.0.0_beta.1-0.x86_64.rpm
./redhat/8/metalk8s-kubernetes-el8/4d300a7655f56307d35f127d99dc192b6aa4997f322234e754f16aaa60fd8906-cri-tools-1.23.0-0.x86_64.rpm
./redhat/8/metalk8s-kubernetes-el8/53edc739a0e51a4c17794de26b13ee5df939bd3161b37f503fe2af8980b41a89-cri-tools-1.12.0-0.x86_64.rpm
./redhat/8/metalk8s-kubernetes-el8/548a0dcd865c16a50980420ddfa5fbccb8b59621179798e6dc905c9bf8af3b34-kubernetes-cni-0.7.5-0.x86_64.rpm
./redhat/8/metalk8s-kubernetes-el8/67ffa375b03cea72703fe446ff00963919e8fce913fbc4bb86f06d1475a6bdf9-cri-tools-1.19.0-0.x86_64.rpm
./redhat/8/metalk8s-kubernetes-el8/79f9ba89dbe7000e7dfeda9b119f711bb626fe2c2d56abeb35141142cda00342-kubernetes-cni-0.5.1-1.x86_64.rpm
./redhat/8/metalk8s-kubernetes-el8/87b6290108e1c8429f6b50895a6376d92f2cd730db2925763c470f32e98b2e2f-kubelet-1.24.6-0.x86_64.rpm
./redhat/8/metalk8s-kubernetes-el8/9165e89a2de0f1a2acfb151177ac6985022ee0c2a8a78d45a4982aa1b11ffd68-cri-tools-1.24.0-0.x86_64.rpm
./redhat/8/metalk8s-kubernetes-el8/ad9e22b62da3b131b3b91fd2caee123cf298133bc58edd6733d7676b697fcd40-kubectl-1.24.6-0.x86_64.rpm
./redhat/8/metalk8s-kubernetes-el8/db7cb5cb0b3f6875f54d10f02e625573988e3e91fd4fc5eef0b1876bb18604ad-kubernetes-cni-0.8.7-0.x86_64.rpm
./redhat/8/metalk8s-kubernetes-el8/e253c692a017b164ebb9ad1b6537ff8afd93c35e9ebc340a52c5bd42425c0760-cri-tools-1.11.0-0.x86_64.rpm
./redhat/8/metalk8s-kubernetes-el8/e382ead81273ab8ebcddf14cc15bf977e44e1fd541a2cfda6ebe5741c255e59f-cri-tools-1.25.0-0.x86_64.rpm
./redhat/8/metalk8s-kubernetes-el8/e7a4403227dd24036f3b0615663a371c4e07a95be5fee53505e647fd8ae58aa6-kubernetes-cni-0.5.1-0.x86_64.rpm
./redhat/8/metalk8s-kubernetes-el8/f70d8cb23c7b91c0509292f4862060367edabce8788b855c38a7c174f014b9e2-cri-tools-1.11.1-0.x86_64.rpm
./redhat/8/metalk8s-kubernetes-el8/fe33057ffe95bfae65e2f269e1b05e99308853176e24a4d027bc082b471a07c0-kubernetes-cni-0.6.0-0.x86_64.rpm
./redhat/8/metalk8s-saltstack-el8/libsodium-1.0.17-2.el8.x86_64.rpm
./redhat/8/metalk8s-saltstack-el8/libunwind-1.3.1-2.el8.x86_64.rpm
./redhat/8/metalk8s-saltstack-el8/openpgm-5.2.122-17.el8.x86_64.rpm
./redhat/8/metalk8s-saltstack-el8/python3-m2crypto-0.33.0-1.el8.x86_64.rpm
./redhat/8/metalk8s-saltstack-el8/python3-msgpack-0.6.1-3.el8.x86_64.rpm
./redhat/8/metalk8s-saltstack-el8/python3-psutil-5.4.3-8.el8.x86_64.rpm
./redhat/8/metalk8s-saltstack-el8/python3-typing-3.5.2.2-4.el8.noarch.rpm
./redhat/8/metalk8s-saltstack-el8/python3-zmq-17.0.0-5.el8.x86_64.rpm
./redhat/8/metalk8s-saltstack-el8/salt-3002.9-1.el8.noarch.rpm
./redhat/8/metalk8s-saltstack-el8/salt-minion-3002.9-1.el8.noarch.rpm
./redhat/8/metalk8s-saltstack-el8/zeromq-4.3.1-4.el8.x86_64.rpm
./redhat/8/metalk8s-scality-el8/x86_64/containerd-1.6.8-1.el8.x86_64.rpm
./redhat/8/metalk8s-scality-el8/x86_64/metalk8s-sosreport-124.0-2.el8.x86_64.rpm
```

List of RPM with this PR
```console
$ find . -name "*.rpm"
./redhat/7/metalk8s-epel-el7/libnetfilter_cthelper-1.0.0-4.el7.x86_64.rpm
./redhat/7/metalk8s-epel-el7/libsodium-1.0.18-1.el7.x86_64.rpm
./redhat/7/metalk8s-epel-el7/python36-asn1crypto-0.24.0-7.el7.noarch.rpm
./redhat/7/metalk8s-epel-el7/python36-cffi-1.9.1-3.el7.x86_64.rpm
./redhat/7/metalk8s-epel-el7/python36-cryptography-2.3-2.el7.x86_64.rpm
./redhat/7/metalk8s-epel-el7/python36-m2crypto-0.35.2-5.el7.x86_64.rpm
./redhat/7/metalk8s-epel-el7/python36-ply-3.9-2.el7.noarch.rpm
./redhat/7/metalk8s-epel-el7/python36-psutil-5.6.7-1.el7.x86_64.rpm
./redhat/7/metalk8s-epel-el7/python36-pycparser-2.14-2.el7.noarch.rpm
./redhat/7/metalk8s-epel-el7/python36-pysocks-1.6.8-7.el7.noarch.rpm
./redhat/7/metalk8s-epel-el7/python36-PyYAML-3.13-1.el7.x86_64.rpm
./redhat/7/metalk8s-epel-el7/python36-rpm-4.11.3-10.el7.x86_64.rpm
./redhat/7/metalk8s-epel-el7/python36-six-1.14.0-3.el7.noarch.rpm
./redhat/7/metalk8s-epel-el7/python36-typing-3.5.2.2-4.el7.noarch.rpm
./redhat/7/metalk8s-kubernetes-el7/14083ac8b11792469524dae98ebb6905b3921923937d6d733b8abb58113082b7-kubernetes-cni-1.1.1-0.x86_64.rpm
./redhat/7/metalk8s-kubernetes-el7/87b6290108e1c8429f6b50895a6376d92f2cd730db2925763c470f32e98b2e2f-kubelet-1.24.6-0.x86_64.rpm
./redhat/7/metalk8s-kubernetes-el7/ad9e22b62da3b131b3b91fd2caee123cf298133bc58edd6733d7676b697fcd40-kubectl-1.24.6-0.x86_64.rpm
./redhat/7/metalk8s-kubernetes-el7/e382ead81273ab8ebcddf14cc15bf977e44e1fd541a2cfda6ebe5741c255e59f-cri-tools-1.25.0-0.x86_64.rpm
./redhat/7/metalk8s-saltstack-el7/openpgm-5.2.122-2.el7.x86_64.rpm
./redhat/7/metalk8s-saltstack-el7/python36-3.6.8-2.el7.x86_64.rpm
./redhat/7/metalk8s-saltstack-el7/python36-chardet-3.0.4-12.el7.noarch.rpm
./redhat/7/metalk8s-saltstack-el7/python36-contextvars-2.4-1.el7.noarch.rpm
./redhat/7/metalk8s-saltstack-el7/python36-distro-1.5.0-1.el7.noarch.rpm
./redhat/7/metalk8s-saltstack-el7/python36-idna-2.7-5.el7.noarch.rpm
./redhat/7/metalk8s-saltstack-el7/python36-immutables-0.14-1.el7.x86_64.rpm
./redhat/7/metalk8s-saltstack-el7/python36-jinja2-2.8.1-3.el7.noarch.rpm
./redhat/7/metalk8s-saltstack-el7/python36-libs-3.6.8-2.el7.x86_64.rpm
./redhat/7/metalk8s-saltstack-el7/python36-markupsafe-0.23-4.el7.x86_64.rpm
./redhat/7/metalk8s-saltstack-el7/python36-msgpack-0.6.2-2.el7.x86_64.rpm
./redhat/7/metalk8s-saltstack-el7/python36-pycurl-7.43.0-8.el7.x86_64.rpm
./redhat/7/metalk8s-saltstack-el7/python36-pyOpenSSL-17.3.0-2.el7.noarch.rpm
./redhat/7/metalk8s-saltstack-el7/python36-requests-2.12.5-4.el7.noarch.rpm
./redhat/7/metalk8s-saltstack-el7/python36-setuptools-39.2.0-4.el7.noarch.rpm
./redhat/7/metalk8s-saltstack-el7/python36-urllib3-1.19.1-6.el7.noarch.rpm
./redhat/7/metalk8s-saltstack-el7/python36-zmq-17.0.0-6.el7.x86_64.rpm
./redhat/7/metalk8s-saltstack-el7/salt-3002.9-1.el7.noarch.rpm
./redhat/7/metalk8s-saltstack-el7/salt-minion-3002.9-1.el7.noarch.rpm
./redhat/7/metalk8s-saltstack-el7/zeromq-4.1.4-7.el7.x86_64.rpm
./redhat/7/metalk8s-scality-el7/x86_64/containerd-1.6.8-1.el7.x86_64.rpm
./redhat/7/metalk8s-scality-el7/x86_64/metalk8s-sosreport-124.0-2.el7.x86_64.rpm
./redhat/8/metalk8s-epel-el8/libsodium-1.0.18-2.el8.x86_64.rpm
./redhat/8/metalk8s-epel-el8/libunwind-1.3.1-3.el8.x86_64.rpm
./redhat/8/metalk8s-epel-el8/openpgm-5.2.122-21.el8.x86_64.rpm
./redhat/8/metalk8s-epel-el8/python3-m2crypto-0.35.2-5.el8.x86_64.rpm
./redhat/8/metalk8s-epel-el8/python3-msgpack-0.6.2-1.el8.x86_64.rpm
./redhat/8/metalk8s-epel-el8/python3-zmq-19.0.0-1.el8.x86_64.rpm
./redhat/8/metalk8s-epel-el8/zeromq-4.3.4-2.el8.x86_64.rpm
./redhat/8/metalk8s-kubernetes-el8/14083ac8b11792469524dae98ebb6905b3921923937d6d733b8abb58113082b7-kubernetes-cni-1.1.1-0.x86_64.rpm
./redhat/8/metalk8s-kubernetes-el8/87b6290108e1c8429f6b50895a6376d92f2cd730db2925763c470f32e98b2e2f-kubelet-1.24.6-0.x86_64.rpm
./redhat/8/metalk8s-kubernetes-el8/ad9e22b62da3b131b3b91fd2caee123cf298133bc58edd6733d7676b697fcd40-kubectl-1.24.6-0.x86_64.rpm
./redhat/8/metalk8s-kubernetes-el8/e382ead81273ab8ebcddf14cc15bf977e44e1fd541a2cfda6ebe5741c255e59f-cri-tools-1.25.0-0.x86_64.rpm
./redhat/8/metalk8s-saltstack-el8/python3-typing-3.5.2.2-4.el8.noarch.rpm
./redhat/8/metalk8s-saltstack-el8/salt-3002.9-1.el8.noarch.rpm
./redhat/8/metalk8s-saltstack-el8/salt-minion-3002.9-1.el8.noarch.rpm
./redhat/8/metalk8s-scality-el8/x86_64/containerd-1.6.8-1.el8.x86_64.rpm
./redhat/8/metalk8s-scality-el8/x86_64/metalk8s-sosreport-124.0-2.el8.x86_64.rpm
```

Diff between before and with this PR
```diff
# diff before now
3d2
< ./redhat/7/metalk8s-epel-el7/openpgm-5.2.122-2.el7.x86_64.rpm
7,9d5
< ./redhat/7/metalk8s-epel-el7/python36-distro-1.5.0-1.el7.noarch.rpm
< ./redhat/7/metalk8s-epel-el7/python36-idna-2.10-1.el7.noarch.rpm
< ./redhat/7/metalk8s-epel-el7/python36-jinja2-2.11.1-1.el7.noarch.rpm
16d11
< ./redhat/7/metalk8s-epel-el7/python36-requests-2.14.2-2.el7.noarch.rpm
20d14
< ./redhat/7/metalk8s-epel-el7/python36-urllib3-1.25.6-2.el7.noarch.rpm
24a19
> ./redhat/7/metalk8s-saltstack-el7/openpgm-5.2.122-2.el7.x86_64.rpm
27a23,24
> ./redhat/7/metalk8s-saltstack-el7/python36-distro-1.5.0-1.el7.noarch.rpm
> ./redhat/7/metalk8s-saltstack-el7/python36-idna-2.7-5.el7.noarch.rpm
28a26
> ./redhat/7/metalk8s-saltstack-el7/python36-jinja2-2.8.1-3.el7.noarch.rpm
33a32
> ./redhat/7/metalk8s-saltstack-el7/python36-requests-2.12.5-4.el7.noarch.rpm
34a34
> ./redhat/7/metalk8s-saltstack-el7/python36-urllib3-1.19.1-6.el7.noarch.rpm
38d37
< ./redhat/7/metalk8s-saltstack-el7/yum-utils-1.1.31-29.el7.noarch.rpm
49,51d47
< ./redhat/8/metalk8s-kubernetes-el8/029bc0d7b2112098bdfa3f4621f2ce325d7a2c336f98fa80395a3a112ab2a713-kubernetes-cni-0.8.6-0.x86_64.rpm
< ./redhat/8/metalk8s-kubernetes-el8/07433570e95a2782cc127e659fe6df434db7f88805e2aed6067768d2f32cb809-cri-tools-1.24.2-0.x86_64.rpm
< ./redhat/8/metalk8s-kubernetes-el8/0c923b191423caccc65c550ef07ce61b97f991ad54785dab70dc07a5763f4222-kubernetes-cni-0.3.0.1-0.07a8a2.x86_64.rpm
53,59d48
< ./redhat/8/metalk8s-kubernetes-el8/14bfe6e75a9efc8eca3f638eb22c7e2ce759c67f95b43b16fae4ebabde1549f3-cri-tools-1.13.0-0.x86_64.rpm
< ./redhat/8/metalk8s-kubernetes-el8/29e7806d1d54cc0eea2963f4ab276778526538816c88c963ece7d1a05fd80792-cri-tools-1.0.0_beta.1-0.x86_64.rpm
< ./redhat/8/metalk8s-kubernetes-el8/4d300a7655f56307d35f127d99dc192b6aa4997f322234e754f16aaa60fd8906-cri-tools-1.23.0-0.x86_64.rpm
< ./redhat/8/metalk8s-kubernetes-el8/53edc739a0e51a4c17794de26b13ee5df939bd3161b37f503fe2af8980b41a89-cri-tools-1.12.0-0.x86_64.rpm
< ./redhat/8/metalk8s-kubernetes-el8/548a0dcd865c16a50980420ddfa5fbccb8b59621179798e6dc905c9bf8af3b34-kubernetes-cni-0.7.5-0.x86_64.rpm
< ./redhat/8/metalk8s-kubernetes-el8/67ffa375b03cea72703fe446ff00963919e8fce913fbc4bb86f06d1475a6bdf9-cri-tools-1.19.0-0.x86_64.rpm
< ./redhat/8/metalk8s-kubernetes-el8/79f9ba89dbe7000e7dfeda9b119f711bb626fe2c2d56abeb35141142cda00342-kubernetes-cni-0.5.1-1.x86_64.rpm
61d49
< ./redhat/8/metalk8s-kubernetes-el8/9165e89a2de0f1a2acfb151177ac6985022ee0c2a8a78d45a4982aa1b11ffd68-cri-tools-1.24.0-0.x86_64.rpm
63,64d50
< ./redhat/8/metalk8s-kubernetes-el8/db7cb5cb0b3f6875f54d10f02e625573988e3e91fd4fc5eef0b1876bb18604ad-kubernetes-cni-0.8.7-0.x86_64.rpm
< ./redhat/8/metalk8s-kubernetes-el8/e253c692a017b164ebb9ad1b6537ff8afd93c35e9ebc340a52c5bd42425c0760-cri-tools-1.11.0-0.x86_64.rpm
66,74d51
< ./redhat/8/metalk8s-kubernetes-el8/e7a4403227dd24036f3b0615663a371c4e07a95be5fee53505e647fd8ae58aa6-kubernetes-cni-0.5.1-0.x86_64.rpm
< ./redhat/8/metalk8s-kubernetes-el8/f70d8cb23c7b91c0509292f4862060367edabce8788b855c38a7c174f014b9e2-cri-tools-1.11.1-0.x86_64.rpm
< ./redhat/8/metalk8s-kubernetes-el8/fe33057ffe95bfae65e2f269e1b05e99308853176e24a4d027bc082b471a07c0-kubernetes-cni-0.6.0-0.x86_64.rpm
< ./redhat/8/metalk8s-saltstack-el8/libsodium-1.0.17-2.el8.x86_64.rpm
< ./redhat/8/metalk8s-saltstack-el8/libunwind-1.3.1-2.el8.x86_64.rpm
< ./redhat/8/metalk8s-saltstack-el8/openpgm-5.2.122-17.el8.x86_64.rpm
< ./redhat/8/metalk8s-saltstack-el8/python3-m2crypto-0.33.0-1.el8.x86_64.rpm
< ./redhat/8/metalk8s-saltstack-el8/python3-msgpack-0.6.1-3.el8.x86_64.rpm
< ./redhat/8/metalk8s-saltstack-el8/python3-psutil-5.4.3-8.el8.x86_64.rpm
76d52
< ./redhat/8/metalk8s-saltstack-el8/python3-zmq-17.0.0-5.el8.x86_64.rpm
79d54
< ./redhat/8/metalk8s-saltstack-el8/zeromq-4.3.1-4.el8.x86_64.rpm
```
</details>

### CODE REVIEW

File: scripts/common.sh.in [hunk 1]
```diff
@@ -279,7 +283,7 @@ configure_salt_minion_local_mode() {
     # Workaround waiting for Rocky Linux support in Salt
     "$SALT_CALL" --file-root="$file_root" \
         --local --retcode-passthrough state.sls \
-	metalk8s.node.rocky-linux-grains saltenv=base
+	    metalk8s.node.rocky-linux-grains saltenv=base
```
Comment (gdemonet) [inline scripts/common.sh.in]:
```suggestion
            metalk8s.node.rocky-linux-grains saltenv=base
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -73,6 +73,16 @@
   image to `alpine:3.16.2`
   (PR[#3888](https://github.com/scality/metalk8s/pull/3888))
 
+### Bug fixes
+
+- Fix an issue in the package availability check from bootstrap script
+  that make it not checking anything
+  (PR[#3898](https://github.com/scality/metalk8s/pull/3898))
+
+- Do no longer include non-needed packages in the MetalK8s internal
```
Comment (gdemonet) [inline CHANGELOG.md]:
```suggestion
- No longer include unnecessary packages in the MetalK8s internal
```

**Review by gdemonet — DISMISSED** [review]
two small nits, otherwise LGTM

File: scripts/common.sh.in (hunk 1 above)
Comment (TeddyAndrieux) [reply scripts/common.sh.in]: Oh it's a tab :cry:

**Review by gdemonet — APPROVED** [review]

**Pull request merged**